database/index.js

```diff
@@ -1,9 +1,9 @@
 const { Sequelize, DataTypes } = require('sequelize');
 
-const sequelize = new Sequelize( 'sbGallery', 'student', '', {
-  // host: 'localhost',
-  host: '54.153.5.233',
-  port: 3306,
+const sequelize = new Sequelize( 'sbGallery', 'student', 'Password!23', {
```

Comment:
You should always be using environment variables that you do not check into github for all passwords, tokens, etc.

Reply:
Updated and will be included in new pull request.